### ISSUE TITLE
ONNX model give this errors how to fix?

### ISSUE THREAD

**Issue by wolfkingal2000** [chaiNNer-org/chaiNNer#2434]
hi went i use ONNX model give this errors how to fix?

Error

An error occurred in a ONNX Upscale Image node:

[ONNXRuntimeError] : 1 : FAIL : Non-zero status code returned while running Concat node. Name:'Concat_22' Status Message: CUDA error cudaErrorNoKernelImageForDevice:no kernel image is available for execution on the device

Input values:
• Image: RGB Image 1400x1995
• Model: Value of type 'nodes.impl.onnx.model.OnnxGeneric'
• Tile Size: 256
• Separate Alpha: No

**Comment by joeyballentine** [comment]
I'm not sure that's something that you or I could fix. What model is it that you are trying to use?

**Comment by wolfkingal2000** [comment]
> I'm not sure that's something that you or I could fix. What model is it that you are trying to use?

It doesn't matter if I want to use any model, that's how it is
I use this site https://openmodeldb.info/models/4x-Nomos8kDAT

**Comment by joeyballentine** [comment]
What GPU do you have?

**Comment by wolfkingal2000** [comment]
> What GPU do you have? 

I use this   graphic  card https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://www.notebookcheck.com/NVIDIA-GeForce-840M.105680.0.html&ved=2ahUKEwjFvtCN5sSDAxVI9LsIHTrjBVwQFnoECBUQAQ&usg=AOvVaw2IKLG-exsRREPn41ahXZS7

**Comment by joeyballentine** [comment]
Make sure you have up to date drivers. Your GPU is kinda old so it might just not support something that the model needs to run. 

Is there a reason you are using the Onnx version of the model and not the pytorch version?

**Comment by wolfkingal2000** [comment]
> Make sure you have up to date drivers. Your GPU is kinda old so it might just not support something that the model needs to run. 
> 
> Is there a reason you are using the Onnx version of the model and not the pytorch version? 

Yes, I have the latest version of drivers for this month and it is cuda version 11.8 and it says correctly about the model, it is not the pytorch version or ok, it does not work either. My problem is that it loads the onnx model but it does not work.

**Comment by joeyballentine** [comment]
So you have tried downloading the pth file and using pytorch? What error did you get when you tried that? I can't help you without more information
